app/sweepers/pseud_sweeper.rb

```diff
@@ -6,19 +6,23 @@ def after_create(record)
   end
   
   def before_update(record)
-    if record.changed.include?(:name)
+    if record.changed.include?("name") || record.changed.include?("login")
       if record.is_a?(User)
-        record.pseuds.each {|pseud| pseud.remove_from_autocomplete}
+        record.pseuds.each {|pseud| pseud.remove_stale_from_autocomplete}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:remove_stale_from_autocomplete` as an argument to `each` instead of a block.